src/csvcubed/definitions.py

```diff
@@ -8,8 +8,10 @@
 import os
 from pathlib import Path
 
+APP_ROOT_DIR_PATH = Path(os.path.dirname(os.path.abspath(__file__)))
```

Comment:
This change makes sure that it is correct when packaged up into a whl (at which point this file is at the root of the package).